src/views/ConfigurationVideoView.vue

```diff
@@ -51,6 +51,29 @@
           />
         </div>
       </div>
+
+      <p class="text-sm font-bold text-grey-darken-1 bg-grey-lighten-5">RTP Jitter Buffer (Target) duration:</p>
+      <div
+        class="flex flex-col items-center px-5 py-3 m-5 font-medium text-center border rounded-md text-grey-darken-1 bg-grey-lighten-5 w-[40%]"
+      >
+        <p>Increase this will result in increased video latency, but can help compensating the network jitter.</p>
+        <br />
+        <p>Cockpit's default is zero milliseconds, but you can leave it empty to use the browser's default.</p>
```

Comment:
```suggestion
        <p> Increasing this will result in increased video latency, but it can help compensate the network jitter.</p>
        <br />
        <p>Cockpit's default is zero milliseconds, but you can leave it empty to use the browser's default.</p>
```